src/hooks/useNotificationTypes.tsx

```diff
@@ -506,10 +511,10 @@ export const notificationTypes: NotificationTypeConfig[] = [
     },
     useNotificationAction: () => {
       const dispatch = useDispatch();
-      const orders = useSelector(getSubaccountOrders, shallowEqual) || [];
+      const orders = useSelector(getSubaccountOrders, shallowEqual) ?? [];
 
       return (orderClientId: string) => {
-        const order = orders.find((order) => order.clientId?.toString() === orderClientId);
+        const order = orders.find((ordr) => ordr.clientId?.toString() === orderClientId);
```

Comment:
[suggestion] `_order` to signify inner/private var while keeping the name more regex-ly equivalent?

Reply:
yeah ordr doesn't look great. I changed it to just "o" since it's a one-liner. I associate variables starting with _ with unused/ignored variables usually, though some use it as private notation too.